ros2netperf/ros2netperf/api/__init__.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2022 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from rclpy.qos import QoSDurabilityPolicy
+from rclpy.qos import QoSHistoryPolicy
+from rclpy.qos import QoSPresetProfiles
+from rclpy.qos import QoSProfile
+from rclpy.qos import QoSReliabilityPolicy
+
+from ros2topic.api import profile_configure_short_keys
+
+
+def positive_int(inval):
+    # Convert first to float, to support scientific notation, like 10e3
+    ret = float(inval)
+    if not ret.is_integer():
+        raise ValueError('Value must be integer')
+    ret = int(ret)
+    if ret <= 0:
+        raise ValueError('Value must be positive')
+    return ret
+
+
+def positive_float(inval):
+    ret = float(inval)
+    if ret <= 0.0:
+        raise ValueError('Value must be positive')
+    return ret
+
+
+def nonnegative_float(inval):
+    ret = float(inval)
+    if ret < 0.0:
+        raise ValueError('Value must be positive or zero')
+    return ret
+
+
+def get_default_qos_profile():
+    return QoSProfile(
+        reliability=QoSReliabilityPolicy.RELIABLE,
+        durability=QoSDurabilityPolicy.TRANSIENT_LOCAL,
+        depth=1)
```

Comment:
Nit: Why not just a constant `_DEFAULT_QOS_PROFILE`?

Reply:
I would have to make a "deep copy" of that constant before modifying the qos profile each time (so the constant is not affected).
I prefer to use a function call directly, so each call is a new object.